flink-test-utils-parent/flink-test-utils/src/test/java/org/apache/flink/state/benchmark/RescalingBenchmark.java

```diff
@@ -16,7 +16,7 @@
  * limitations under the License
  */
 
-package org.apache.flink.contrib.streaming.state.benchmark;
+package org.apache.flink.state.benchmark;
```

Comment:
Have you prepared the changes on the flink-benchmarks side? we should minimize the time flink-benchmarks compilation fails.

Reply:
Sure. PR Link: https://github.com/apache/flink-benchmarks/pull/75